var/spack/repos/builtin/packages/hdf-eos2/package.py

```diff
@@ -75,6 +83,18 @@ def url_for_version(self, version):
                 "version/checksum not found in version_list".format(version)
             )
 
+    @run_before("configure")
+    def fix_permissions(self):
```

Comment:
Can you add a comment on why this is needed? Hopefully we can get rid of it in newer versions.

Reply:
Done.